### DISCUSSION TITLE
navigation bar got bigger, can't fix it now

### DISCUSSION THREAD

**Discussion by Rust-soham** [excalidraw/excalidraw#8775] [General]
was zooming in and out of my canvas, suddenly the nav bar got bigger and it can't seem to go back now.

![Image](https://github.com/user-attachments/assets/f47c31e7-a842-430c-87bc-4e75b22451a4)

**Comment by dwelle** [comment]
Hi, it's likely your browser is zoomed. Check the right corner in your url address bar for a magnifying glass.
